### PR TITLE
feat: upgrade oa for DID signed document

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "babel-jest": "^25.2.3",
     "babel-loader": "8.0.6",
     "brotli-webpack-plugin": "^1.1.0",
-    "compression-webpack-plugin": "^6.0.3",
+    "compression-webpack-plugin": "^6.0.2",
```
Comment (Nebulis) [inline package.json]: why downgrade ?

File: src/services/verify/fragments.tsx [hunk 2]
```diff
@@ -24,11 +24,17 @@ export const isValid = (
   types: VerificationFragmentType[] = ["DOCUMENT_STATUS", "DOCUMENT_INTEGRITY", "ISSUER_IDENTITY"]
 ) => {
   if (types.includes("ISSUER_IDENTITY")) {
-    const dnsFragment = getFirstFragmentFor(verificationFragments, "OpenAttestationDnsTxt");
-    return (
-      isValidFromUpstream(verificationFragments, types) &&
-      dnsFragment?.data?.every((issuer: VerificationFragment) => issuer.status === "VALID")
-    );
+    const dnsTxtFragment = getFirstFragmentFor(verificationFragments, "OpenAttestationDnsTxt");
```
Comment (Nebulis) [inline src/services/verify/fragments.tsx]: Maybe can do something like:
```
const identityFragments = fragments.filter(fragment => fragment.type === "ISSUER_IDENTITY" && fragment.status === "VALID");
const allIdentityValid = identityFragments.some(fragment => fragment?.data?.every((issuer: VerificationFragment) => issuer.status === "VALID"))
```

File: src/services/verify/fragments.tsx [hunk 3]
```diff
@@ -24,11 +24,17 @@ export const isValid = (
   types: VerificationFragmentType[] = ["DOCUMENT_STATUS", "DOCUMENT_INTEGRITY", "ISSUER_IDENTITY"]
```
Comment (Nebulis) [inline src/services/verify/fragments.tsx:25]: there are no tests

**Review by Nebulis — APPROVED** [review]

**Comment by rjchow** [comment]
closed by #276

**Pull request closed without merging**